docs/examples/robust_prod_inventory.ipynb

```diff
@@ -0,0 +1,335 @@
+    "$$\n",
+    " \\mathcal{U}_{\\text{Polyhedral}} = \\{Az+b \\ | \\ \\|z \\|_p \\le \\rho\\}\n",
+    "$$\n",
+    "The following code defines these parameters for the problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def setup_parameters(proportion):\n",
+    "    P = np.full((I, T), MAX_CAPACITY)\n",
+    "    Q = [INTEGRAL_CAPACITY] * I\n",
+    "    alphas = np.array([1, 1.5, 2])\n",
+    "    c = [(alphas * (1 + HALF * np.sin(np.pi * (t - 1) / (T * HALF)))).flatten() for t in range(1, T + 1)]\n",
+    "    c = np.array(c).T\n",
+    "    \n",
+    "    t_values = np.arange(1, T + 1)\n",
+    "    d_star = SEASONAL_MULTIPLIER * (1 + HALF * np.sin(np.pi * (t_values - 1) / (T * HALF)))\n",
+    "    \n",
+    "    eye_matrix = np.eye(T)\n",
+    "    neg_eye_matrix = -np.eye(T)\n",
+    "    lhs = np.concatenate((eye_matrix, neg_eye_matrix), axis=0)\n",
+    "    \n",
+    "    rhs_upper = (1 + proportion) * d_star\n",
+    "    rhs_lower = (-1 + proportion) * d_star\n",
+    "    rhs = np.hstack((rhs_upper, rhs_lower))\n",
+    "    \n",
+    "    d = lropt.UncertainParameter(T, uncertainty_set=lropt.Polyhedral(lhs=lhs, rhs=rhs))\n",
+    "    \n",
+    "    return P, Q, c, d, rhs_upper, rhs_lower, d_star\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we create a function to define the constraints of the function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def define_constraints(P, Q, c, d):\n",
+    "    F = cp.Variable(nonneg=True)\n",
+    "    p = cp.Variable((I, T), nonneg=True)\n",
+    "\n",
+    "    constraints = [\n",
+    "        cp.sum(cp.multiply(c, p)) <= F,\n",
+    "        p <= P,\n",
+    "        cp.sum(p, axis=1) <= Q\n",
+    "    ]\n",
+    "    \n",
+    "    for i in range(1, T + 1):\n",
+    "        mult = np.hstack([np.ones(i), np.zeros(T - i)])\n",
+    "        constraints.append(V_MIN <= cp.sum(cp.sum(cp.sum(p, axis=0)[:i])) - mult @ d + V_INITIAL)\n",
+    "        constraints.append(cp.sum(cp.sum(cp.sum(p, axis=0)[:i])) - mult @ d + V_INITIAL <= V_MAX)\n",
+    "\n",
+    "    return F, p, constraints"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is a function that solves the problem and returns the optimal value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def solve_problem(F, p, constraints):\n",
+    "    objective = cp.Minimize(F)\n",
+    "    prob = lropt.RobustProblem(objective, constraints)\n",
+    "    prob.solve()\n",
+    "    return F.value, p.value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following code calculates the robust optimal level of inventory over time. "
```

Comment:
For the visualization, can you also generate a random trajectory, for both perturbation = 0.1 and 0.2. The inventory levels for that trajectory should be between the current upper and lower bounds.